integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/LuceneTestCase.scala

```diff
@@ -110,14 +110,13 @@ class LuceneTestCase extends QueryTest with BeforeAndAfterAll {
     checkAnswer(sql("SELECT * FROM datamap_main WHERE TEXT_MATCH('country:ch*')"),
       sql("select * from datamap_main where country like 'ch%'"))
     checkAnswer(sql(
-      "SELECT * FROM datamap_main WHERE TEXT_MATCH('country:ch*') AND TEXT_MATCH('name:aa*')"),
+      "SELECT * FROM datamap_main WHERE TEXT_MATCH('country:ch* AND name:aa*')"),
       sql("select * from datamap_main where country like 'ch%' and name like 'aa%'"))
     checkAnswer(sql(
-      "SELECT * FROM datamap_main WHERE TEXT_MATCH('country:u* or name:aa*')"),
+      "SELECT * FROM datamap_main WHERE TEXT_MATCH('country:u* OR name:aa*')"),
```

Comment:
you mean to say `country:u* or name:aa*` is different from `country:u* OR name:aa*`?

Reply:
yes..For lucene -standard analyzer ,filters should be given in UPPER case